samples/stock/demo/stock-css-design/demo.js

```diff
@@ -0,0 +1,131 @@
+    const data = await fetch(
+        'https://demo-live-data.highcharts.com/aapl-ohlcv.json'
+    ).then(response => response.json());
+
+    // split the data set into ohlc and volume
+    const ohlc = [],
+        volume = [],
+        dataLength = data.length;
+
+    for (let i = 0; i < dataLength; i += 1) {
+        ohlc.push([
+            data[i][0], // the date
+            data[i][1], // open
+            data[i][2], // high
+            data[i][3], // low
+            data[i][4] // close
+        ]);
+
+        volume.push([
+            data[i][0], // the date
+            data[i][5],
+            data[i][1] < data[i][4] ? 'highcharts-point-up' : 'highcharts-point-down' // the volume
+        ]);
+    }
+
+    Highcharts.stockChart('container', {
+        xAxis: {
+            crosshair: {
+                className: 'highcharts-crosshair-custom',
+                enabled: true
+            }
+        },
+        yAxis: [{
+            crosshair: {
+                snap: false,
+                className: 'highcharts-crosshair-custom',
+                label: {
+                    className: 'highcharts-crosshair-custom-label',
+                    enabled: true,
+                    format: '{value:.2f}'
+                },
+                enabled: true
+            },
+            labels: {
+                align: 'left'
+            },
+            height: '70%'
+        }, {
+            crosshair: {
+                className: 'highcharts-crosshair-custom',
+                snap: false,
+                enabled: true,
+                label: {
+                    format: '{#if value ge 1000000} {(divide value 1000000):.2f} M {else} {value} {/if}',
+                    className: 'highcharts-crosshair-custom-label',
+                    enabled: true
+                }
+            },
+            labels: {
+                align: 'left'
+            },
+            top: '70%',
+            height: '30%',
+            offset: 0
+        }],
+        chart: {
+            styledMode: true
+        },
+        tooltip: {
+            shape: 'square',
+            split: false,
+            shared: true,
+            headerShape: 'callout',
+            shadow: false,
+            formatter: function () {
+                const point = this.points[0].point;
+                const color = point.open > point.close ? '#ff4242' : '#51af7b';
+                // @eslint-ignore max-len
+                return `<span style="font-size: 1.4em">  ${this.series.name}</span> O<span style="color: ${color}";>${point.open}</span> H<span style="color: ${color}";>${point.high}</span> L<span style="color: ${color}";>${point.low}</span> C<span style="color: ${color}";>${point.close}</span> <span style="color: ${color}";>${Math.floor(point.open - point.close)} (${Math.floor((point.open - point.close) / point.close * 100) / 100}%)</span><br>Volume <span style="color: ${color}";>${this.points[1].point.y}</span>`;
+            },
```

Comment:
We can avoid the formatter and keep the config 100% serializable by using this logic for the color: 
`{#ge point.open point.close}#ff4242{else}#51af7b{/ge}`

It's a little bit verbose and must be repeated for each line, but it works.

Also try to break the lines at 80 characters, otherwise this will be hard to read for example in jsFiddle. Backticks are okay.

Reply:
That's a lot of templating 😅 But can do 👍